packit_service/worker/fedmsg_handlers.py

```diff
@@ -141,6 +141,20 @@ def __init__(self, config: ServiceConfig, job: JobConfig, event: CoprBuildEvent)
         self.project = self.event.get_project()
         self.package_config = self.event.get_package_config()
 
+    def was_last_build_successful(self):
+        """
+        Check if the last copr build of the PR was succesful
```

Comment:
I think the method actually checks whether ANY (not last) build was successful.

Reply:
It goes through the comments in reverse order and picks the first with `packit-as-a-service` as an author. If it describes success, it returns `True` and `False` otherwise and also `False` when there is no such comment.

I don't see the problem.

Reply:
you're right, sorry for the fuss